doc/README.open_ce_test.md

```diff
@@ -0,0 +1,136 @@
+# README for `open-ce test` command
+
+* [`open-ce test env`](#open-ce-test-env-sub-command)
+* [`open-ce test feedstock`](#open-ce-test-feedstock-sub-command)
+
+## Open-CE Tests
+
+Open-CE provides tools for running tests for packages built by the Open-CE tools. To add tests to a feedstock, create a file named `tests/open-ce-tests.yaml`. The test can include a list of tests that will be run within a newly generated conda environment that should include the packages being built by the feedstock.
+
+### Open-CE Test File
+
+An Open-CE Test file is a YAML file that describes the tests that will be run. The test files can also include jinja code. Below is a simple example of a test file:
+
+```yaml
+tests:
+  - name: First Test
+    command: |
+      cd my_package
+      pytest -v tests
+{% if build_type == 'cuda' %}
+  - name: CUDA Test
+    command: |
+      cd my_package
+      pytest -v cuda_tests
+{% endif %}
+```
+
+This example will:
+
+1. Create a new conda environment that will be active throughout the running of the tests.
+1. Run the bash instructions specified by `command` in "First Test" within a new bash shell that activates the conda environment from step 1.
```

Comment:
Numbering needs correction.

Reply:
This should be rendered correctly. I believe that this is the proper way to write a numbered list in Markdown.